frontend/src/pages/notebookController/screens/server/ImageTagPopover.tsx

```diff
@@ -35,7 +35,14 @@ const ImageTagPopover: React.FC<ImageTagPopoverProps> = ({ tag, description }) =
         </>
       }
     >
-      <OutlinedQuestionCircleIcon />
+      <Button
```

Comment:
I know `Button` could create some paddings which will break the layout so you need to add custom CSS to override it. We can also consider the `Icon` component.
Maybe we can do the following:
```
<Icon aria-label="More info" role="button" tabIndex={0}>
<OutlinedQuestionCircleIcon />
</Icon>
```
I tested this on PF examples and it works for VO.

Reply:
Thanks @DaoDaoNoCode - great idea